session/bootstrap.go

```diff
@@ -2079,6 +2093,15 @@ func upgradeToVer103(s Session, ver int64) {
 	doReentrantDDL(s, CreateStatsTableLocked)
 }
 
+func upgradeToVer104(s Session, ver int64) {
+	if ver >= version104 {
+		return
+	}
+	doReentrantDDL(s, CreatePasswordHistory)
+	doReentrantDDL(s, "Alter table mysql.user add COLUMN IF NOT EXISTS `Password_reuse_history` smallint unsigned  DEFAULT NULL  AFTER `Create_Tablespace_Priv` ")
```

Comment:
```suggestion
	doReentrantDDL(s, "Alter table mysql.user add COLUMN IF NOT EXISTS `Password_reuse_history` smallint unsigned  DEFAULT NULL AFTER `Create_Tablespace_Priv` ")
```
Please remove the blank line.